package/kedro_viz/data_access/repositories/catalog.py

```diff
@@ -4,11 +4,17 @@
 import logging
 from typing import Dict, Optional
 
-from kedro.io import AbstractDataSet, DataCatalog, DatasetNotFoundError, MemoryDataset
+from kedro.io import AbstractDataSet, DataCatalog
```

Comment:
Ahead of https://github.com/kedro-org/kedro/pull/2746, put these in a try-except block too.

See for reference: https://github.com/kedro-org/kedro-plugins/pull/255/files#diff-b3a12c160c707dc01220f1964faa902900dbcac620993d3fd0fdae9c8787797f

Reply:
It should not be in the same try-except block though, because they weren't deprecated in the same version.